sdk-core/transaction.md

```diff
@@ -0,0 +1,46 @@
+## Transaction
+
+```
+class Transaction:
```

Comment:
another approach here will be the following: use a wrapper over the FrontEndTransaction as defined here: https://github.com/multiversx/mx-chain-core-go/blob/main/data/transaction/frontendTransaction.go
This will be a plain DTO with all fields exported. Then we can create a "transaction interactor" that manipulates frontend transaction DTOs and provide additional functions like sign, guardian sign and so on.
Reason: we can easily start with a partially filled transaction pointer and pass that pointer on different "objects" that manipulates certain fields (maybe one will populate the gas price& chain ID, the other ones might manipulate the Data and gaslimit)
Also, this will avoid having a constructor with a lots of parameters.

Reply:
Sounds all right for Go, yes. For other SDKs, we have to write it "from scratch".

Reply:
About transaction interactors - to be discussed.